internal/sidekick/rust/templates/common/client_method_samples/builder_fields.mustache

```diff
@@ -13,4 +13,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 }}
-///         /* set fields */
+{{#StandardGetInfo}}
+///         .set_{{ResourceNameRequestField.Codec.SetterName}}(resource_name)
+{{/StandardGetInfo}}
+{{^IsStandardMethod}}
+///         .with_request(request)
```

Comment:
Hmmm.. probably not the direction we want to steer folks into. Yes, it works, but it is the least desirable way to initialize a request.  Can we leave these complex requests as `/* set fields */`  for now?